front/cypress/e2e/admin/pages_and_menu/create_update_and_view_custom_page.cy.ts

```diff
@@ -245,11 +245,14 @@ describe('Admin: can', () => {
 
       cy.get('#local_page_files').should('exist');
       cy.wait('@getFiles');
+
+      cy.wait(2000);
```

Comment:
I know it's not the most elaborate solution but somehow this fixes the flakiness for me locally. Let's see what happens on production